### PR TITLE
Merge 2dot3 into master

### PR DESCRIPTION
**Pull request by jjanvier**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: .ci/Jenkinsfile [hunk 1]
```diff
@@ -112,20 +117,20 @@ stage("Test") {
             )},
             "front-acceptance-ee": {testif(
                 condition: launchAcceptanceTests.equals("yes") && editions.contains("ee"),
-                container: tag,
+                container: "${tag}-ee",
                 script: "cd /var/www/pim && yarn run webpack-test && MAX_RANDOM_LATENCY_MS=100 yarn run acceptance ./vendor/akeneo/pim-community-dev/tests/features ./tests/features"
             )},
             "back-acceptance-ce": {queue(
```
Comment (damien-carcel) [inline .ci/Jenkinsfile]: no, it should be a `testif`

File: .ci/Jenkinsfile [hunk 2]
```diff
@@ -112,20 +117,20 @@ stage("Test") {
...
                 condition: launchAcceptanceTests.equals("yes"),
                 verbose: (verboseOutputs == "yes"),
                 container: tag,
-                fetcher: {return pimBehatFetcher(profiles: ["acceptance"], features: "tests/features")}
+                script: "cd /var/www/pim && vendor/bin/behat --strict -p acceptance -vv"
             )},
             "back-acceptance-ee": {queue(
```
Comment (damien-carcel) [inline .ci/Jenkinsfile]: testif

File: features/mass-action/add_products_and_variant_products_to_groups.feature [hunk 3]
```diff
@@ -0,0 +1,61 @@
+@javascript
```
Comment (damien-carcel) [inline features/mass-action/add_products_and_variant_products_to_groups.feature]: should be moved in tests/legacy

**Comment by damien-carcel** [comment]
Basically, only legacy behats and backend integration tests should be run with the queue. Anything else should use a `testif`.

File: .ci/Jenkinsfile (hunk 1 above)
Comment (jjanvier) [reply .ci/Jenkinsfile]: fixed

File: .ci/Jenkinsfile (hunk 2 above)
Comment (jjanvier) [reply .ci/Jenkinsfile]: fixed

**Comment by jjanvier** [comment]
Only one Behat failing in CE.
Fixed by the last commit.

**Pull request merged**